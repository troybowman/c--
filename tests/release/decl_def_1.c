../decl/decl_def_1.c