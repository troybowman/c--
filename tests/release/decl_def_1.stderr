error: function dont_define_me is defined at line 5 but is declared extern at line 2
error: function dont_define_me redefined at line 6 (previous definition starts at line 5)
error: non-void funcion dont_define_me must return a value
c-- returned nonzero exit code